nautobot/extras/urls.py

```diff
@@ -552,6 +553,19 @@
         views.RelationshipAssociationDeleteView.as_view(),
         name="relationshipassociation_delete",
     ),
+    # Roles
+    path(
+        "roles/<str:slug>/changelog/",
+        views.ObjectChangeLogView.as_view(),
+        name="role_changelog",
+        kwargs={"model": Role},
+    ),
+    path(
+        "roles/<str:slug>/notes/",
+        views.ObjectNotesView.as_view(),
+        name="role_notes",
+        kwargs={"model": Role},
+    ),
```

Comment:
These will go away once converting the Role views to a NautobotUIViewSet, right?

Reply:
I think `Role` views are already using `NautobotUIViewSet`. But `ChangeLogMixin` and `NotesMixin` are not in the ViewSet on this branch for some reason. Maybe this branch is not up to date with the base branch?

Reply:
Nope, NautobotUIViewSet do not account for object_change_log and notes

Reply:
It should, I verified that was working with the CircuitsViewSet

Reply:
Oh, looks like the Role model doesn't have the notesmixin, as far as I can see.

Reply:
The NautobotUIViewSetRouter should have the Urls in it.

Reply:
Thats for api, This is UI

Reply:
nautobot.circuits.urls.py:L55
```python
path(
        "circuits/<uuid:pk>/changelog/",
        ObjectChangeLogView.as_view(),
        name="circuit_changelog",
        kwargs={"model": Circuit},
    ),
    path(
        "circuits/<uuid:pk>/notes/",
        ObjectNotesView.as_view(),
        name="circuit_notes",
        kwargs={"model": Circuit},
    ),
```

Reply:
Weird, my PR has it removed. I will send it when I find the PR on the computer, can't link the lines from my phone. This is most definitely not just the API, and there might have been a regression.

Reply:
In addition, these are the old views, not the ViewSets.

Reply:
Check out the develop branch, it does not have those urls.

Reply:
oooh True, Illd have to update my branch. Thanks